src/types/typeobject.rs

```diff
@@ -127,6 +130,16 @@ pub trait PyTypeMethods<'py>: crate::sealed::Sealed {
     fn is_subclass_of<T>(&self) -> PyResult<bool>
     where
         T: PyTypeInfo;
+
+    /// Return Python mro
+    ///
+    /// Equivalent to the Python expression `__mro__`.
+    fn mro(&self) -> PyResult<Bound<'_, PyTuple>>;
+
+    /// Return Python bases
+    ///
+    /// Equivalent to the Python expression `__bases__`.
```

Comment:
Same question where I'm wondering about linking to the docs.

```suggestion
    /// Equivalent to the Python expression `self.__bases__`.
```